src/jaxsim/mujoco/loaders.py

```diff
@@ -598,21 +611,106 @@ def convert(
 
 @dataclasses.dataclass
 class MujocoCamera:
-    name: str
-    mode: str
-    pos: str
-    xyaxes: str
-    fovy: str
+
+    mode: str = "fixed"
+
+    target: str | None = None
+    fovy: str = "45"
+    pos: str = "0 0 0"
+
+    quat: str | None = None
+    axisangle: str | None = None
+    xyaxes: str | None = None
+    zaxis: str | None = None
+    euler: str | None = None
+
+    name: str | None = None
```

Comment:
Should we somehow specify that the usage of some attributes excludes the usage of other attributes?

Reply:
I guess that we can redirect users to the Mujoco documentation as already done elsewhere in this PR. I don't want to dedicate any time to copy-paste complex documentation. Does it sound good?

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/206/commits/6b86a1527072578eca4a26863b45526bf7aef228.